test/ed25519-test.js

```diff
@@ -48,7 +47,7 @@ describe('sign.input ed25519 test vectors', function() {
 
   before(function(done) {
     ed25519 = new eddsa('ed25519');
-    fs.readFile(__dirname + '/fixtures/sign.input', function(err, f) {
+    require('fs').readFile(__dirname + '/fixtures/sign.input', function(err, f) {
```

Comment:
this instead is needed for 'brfs' that requires inline syntaxes to do the magic.